bgpd/bgp_filter.c

```diff
@@ -511,7 +511,7 @@ DEFUN(no_as_path, no_bgp_as_path_cmd,
 	bgp_regex_free(regex);
 
 	if (asfilter == NULL) {
-		vty_out(vty, "\n");
+		vty_out(vty, "filter doesn't exist\n");
```

Comment:
Can we change this to something like this: `Regex entered %s does not exist`?

Reply:
Done.